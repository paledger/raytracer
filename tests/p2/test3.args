pixelcolor ../resources/simple.pov 640 480 400 105
