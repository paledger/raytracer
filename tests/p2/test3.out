Pixel: [400, 105] Ray: {0 0 14} -> {0.1594 -0.2664 -0.9506}
T = 15.02
Object Type: Plane
BRDF: Blinn-Phong
Color: (56, 56, 225)
